doc/src/errorrecovery.md

```diff
@@ -253,9 +254,9 @@ few lines of code.
 ## Making use of `%epp` for easier to read repair sequences
 
 By default, pretty-printing lexeme types prints out their identifier in the
-grammar. To this point these have matched the output the user would expect,
-this is rarely always the case though.  If we used word identifiers instead of symbols
-as given by the following calc.l:
+grammar. Up until this point lexeme identifiers and the output has matched the output
```

Comment:
I can't really parse the "up until" sentence. Maybe "Up to now, we have used lexeme types when showing output to the user. While the lexeme types are sometimes adequate for this purpose, this is not always the case. Consider this lex file:"?

Reply:
Fixed that in [9e8b8e9](https://github.com/softdevteam/grmtools/pull/366/commits/9e8b8e9142500a82f4f9ebf5369619ec40778ae4) along another small change to the next sentence.